.oca-port.json

```diff
@@ -0,0 +1,11 @@
+{
+  "14.0": {
+    "16.0-mig-stock_available_to_promise_release": {
+      "stock_available_to_promise_release": {
+        "blacklist_pull_requests": {
+          "orphaned_commits": true
+        }
+      }
+    }
+  }
+}
```

Comment:
This file has to be removed, it was generated by the first release of `oca-port`, it has been refactored in the latest release and will store things in `.oca/` folder.